templates/taxi/car_detail.html

```diff
@@ -18,5 +18,16 @@ <h1>Drivers</h1>
     {% for driver in car.drivers.all %}
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
+  {% if user in car.drivers.all %}
+  <form method="post" action="{% url 'car-remove-driver' car.pk user.pk %}">
+    {% csrf_token %}
+    <button type="submit" class="btn btn-danger">Delete me from this car</button>
+  </form>
+{% else %}
+  <form method="post" action="{% url 'car-assign-driver' car.pk user.pk %}">
+    {% csrf_token %}
+    <button type="submit" class="btn btn-primary">Assign me to this car</button>
+  </form>
+{% endif %}
```

Comment:
Ensure that the URLs 'car-remove-driver' and 'car-assign-driver' are correctly defined in your `urls.py` file. These URLs should correspond to views that handle the logic for assigning and removing drivers from cars.